extension/src/SidebarProvider.ts

```diff
@@ -107,34 +107,12 @@ export class SidebarProvider implements vscode.WebviewViewProvider {
           // use getConfiguration to check what the current settings are for the user
           const settings = await vscode.workspace.getConfiguration('rct');
           // send a message back to the webview with the data on settings
-          webviewView.webview.postMessage({
+          await webviewView.webview.postMessage({
             type: "settings-data",
             value: settings.view
           });
           break;
         }
-
-        // Case when sapling becomes visible in sidebar
```

Comment:
@MajorLift  is it okay to remove this ? or is it handled somewhere else ?

Reply:
These two cases are duplicates of lines 95-115 right above (with the only difference being the added `await` keywords), and results of a merge conflict I handled wrong in the first PR. Removing them should be safe and necessary.